Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2852,16 +2854,17 @@ extension BrowserViewController {
          Specific values here are from the WKContentView headers.
          https://github.com/JaviSoto/iOS9-Runtime-Headers/blob/master/Frameworks/WebKit.framework/WKContentView.h
         */
-        guard let webContentView = UIView.findSubViewWithFirstResponder(webView),
-            let input = webContentView.performSelector(Selector("inputAccessoryView")),
+        guard let webContentView = UIView.findSubViewWithFirstResponder(webView) else {
+            return
```

Comment:
I wouldn't want to crash the browser just because we couldn't find the webview to add a searchButton. I think its better if the feature just silently failed.